index.html

```diff
@@ -2564,11 +2564,11 @@ <h2>Definition of Roles</h2>
 			</table>
 		</div>
 		<div class="role" id="definition">
-			<rdef>definition</rdef>
-			<div class="role-description">
-				<p>A definition of a term or concept. See related <rref>term</rref>.</p>
-				<p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-labelledby</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
-			</div>
+      <rdef>definition</rdef>
+      <div class="role-description">
+        <p>A definition of a term or concept. See related <rref>term</rref>.</p>
+        <p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-details</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
```

Comment:
I worry this sentence could be interpreted to say authors should put the `aria-details` on the `role=definition` element, rather than on the `role=term` element and have it point to the definition's `id`.

would something like this text work (adjust further as necessary), and do we want to add a quick code snippet?

>Authors SHOULD identify the element being defined by giving it a role of `term`. The `term` should then reference the `definition` with the `aria-details` attribute.  Alternatively, a `term` may be a descendant of the element with role `definition`, to convey its association.

```html
<span role=term aria-details=def>some word</span> 
<span role=definition id=def>some definition...</span>

<!-- or  -->

<span role=definition>
  <span role=term>some word</span> some definition...
</span>
```

Reply:
I think you're right. I'm updating the spec text for both term and definition. In addition to clarifying, I'm removing the option to use the descendant approach. It really doesn't make any sense to have one be the descendant of the other, and won't lead to a good or consistent AT experience.